superset/config.py

```diff
@@ -170,10 +170,15 @@ def _try_json_readsha(filepath):
 # [TimeZone List]
 # See: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
 # other tz can be overridden by providing a local_config
-DRUID_IS_ACTIVE = True
 DRUID_TZ = tz.tzutc()
 DRUID_ANALYSIS_TYPES = ["cardinality"]
 
+# Legacy Druid connector
+# Druid supports a SQL interface in its newer versions.
+# Setting this flag to True enables the deprecated, API-based Druid
+# connector. This feature may be removed at a future date.
+DRUID_IS_ACTIVE = False
```

Comment:
@willbarret apologies for not mentioning this sooner but can we rename this to something more specific, i.e., `DRUID_NATIVE_IS_ACTIVE` or `DRUID_NOSQL_IS_ACTIVE`?